### PR TITLE
Postcss

### PR DESCRIPTION
**Pull request by jrief**
@AndreasBackx could you please review this

### CODE REVIEW

**Comment by AndreasBackx** [comment]
Looks good to me, but I'd love to see a setting to disable PostCSS as it's something that can take a while for large projects and is not really necessary during development. The setting can default to `not DEBUG`.

Regarding whether we should actually implement these things is another matter. For a recent project I've been using [django-compressor-postcss](https://github.com/Pithikos/django-compressor-postcss) and now I've also been thinking about using django-compressor directly as it offers sass support as well. But I haven't really looked into it myself too much. I'm also in the middle of my finals for university so don't have much time for it right now either unfortunately.

**Comment by jrief** [comment]
@AndreasBackx I now added a configuration directive to disable that feature.

There are two reasons I still prefer django-sass-processor over a pure post-compiling step:

1. Inside my Django templates I have to refer to the source file (ie. `xyz.scss`) rather than referring onto the final hence compiled file (ie. `xyz.css`). This gives a much straighter information to developers new to a project.

2. I can reuse configuration directives from my `settings.py` directly inside my SCSS files. This is handy for specifying search paths for fonts, etc. AFAIK, you can't do that with django-compressor.

**Comment by AndreasBackx** [comment]
@jrief 

1. I've read in the documentation that you can add your own MIME types and then directly use the file itself rather than the compiled file.
2. I have not found that need myself yet, could you give an example?

Regarding my review and the like. I'm studying for my finals and am on holiday after so I cannot be active on GitHub the coming 2 months unfortunately. I'll be free starting from around mid-February I believe. I really just can't afford any distractions during this time.

File: README.md [hunk 1]
```diff
@@ -230,6 +237,35 @@ class SomeAdminOrFormClass(...):
...
+## Add vendor prefixes to CSS rules using values from https://caniuse.com/
+
+Writing SCSS shall be fast and easy and you should not have to care, whether to add vendor specific
+prefixes to your CSS directives. Unfortunately there is no pure Python package to solve this, but
+with a few node modules, we can add this to our process chain.
+
+As superuser install
+
+```shell
+npm install -g npx
```
Comment (AndreasBackx) [inline README.md]: Shouldn't we recommend using Yarn instead? Then the command is `yarn global add npx`.

File: sass_processor/processor.py [hunk 2]
```diff
@@ -107,7 +124,7 @@ def is_latest(self, sourcemap_filename):
         if not sourcemap_file or not os.path.isfile(sourcemap_file):
             return False
         sourcemap_mtime = os.stat(sourcemap_file).st_mtime
-        with open(sourcemap_file, 'r') as fp:
+        with io.open(sourcemap_file, 'r') as fp:
```
Comment (AndreasBackx) [inline sass_processor/processor.py]: Do we really need to use the `io` module here?

File: README.md (hunk 1 above)
Comment (jrief) [reply README.md]: I have no opinion on this. I use ``npm`` for all of my projects and had no problems with it up to now. It what respect is ``yarn`` better?

File: sass_processor/processor.py (hunk 2 above)
Comment (jrief) [reply sass_processor/processor.py]: No, I'll remove it.

File: README.md (hunk 1 above)
Comment (AndreasBackx) [reply README.md]: I've heard some negative thing related to the development of npm. Besides that yarn also uses more concise lockfiles and checksums, but npm could've also implemented this by now. I'm not a real fan of Javascript, but Yarn seems to be what everyone is transitioning towards. It's also faster in comparison from what I read online.

**Comment by AndreasBackx** [comment]
LGTM, I'd say squash and merge the branch when everything is okay.

**Pull request merged**